__tests__/app.test.js

```diff
@@ -45,8 +45,18 @@ describe("All endpoints", () => {
 				})
 		})
 	})
-	//-----#14 endpoint ----------
+	//-----#14 GET api/articles/:article_id endpoint controller -----
 	describe("GET - /api/articles/:article_id", () => {
+		// returns a comment count
+		test("Status 200, responds with comment count ", () => {
+			return request(app)
+				.get("/api/articles/1")
+				.expect(200)
+				.then(({ body: { article } }) => {
+					console.log(article, "<<<<<")
+					expect(article.comment_count).toBe("11")
```

Comment:
This test isn't adding much more value than the test you already have in place below

You can be more specific in your test below and expect the value to be `"11"`

Reply:
The reason I did this test is because in the test below I initially had the test out put the specific return, however when I did a pull request I was told to refactor and test for properties rather than specific. hence I added this test to make sure count was working